global.json

```diff
@@ -27,5 +27,8 @@
     "src/ResourceManagement/Insights/Microsoft.Azure.Insights",
     "src/ResourceManagement/AnalysisServices/Microsoft.Azure.Management.Analysis",
     "src/ResourceManagement/Monitor/Microsoft.Azure.Monitor"
-  ]
+  ],
+  "sdk": {
```

Comment:
I have VS2017 installed on my machine, without this the build is totally broken. See https://github.com/Azure/azure-sdk-for-net/issues/2922 .